src/index.js

```diff
@@ -75,17 +46,18 @@ const actions = {
     return {
       ...state,
       selectedNode,
-      showEditor: false,
+      showEditor: true,
     }
   },
 
   onGraphDblClick: ev => (state, actions) => {
     logger.log('graph double clicked', ev)
+    state.graphData = retrieveGraphData(ev.nodeId)
```

Comment:
I can see what you're doing here, but just a thing to keep in mind, things are to be treated as immutable here. You're assigning a new value to an object. Instead just assign the value to variable, then in the `return` statement, just do `graphData: newGraphData` or something.

You don't want to start mutating data in this way of writing interfaces (it's fine for angular which is OOP and encourages mutable state), but react and the like encourage immutability. It's easy to accidentally mutate state and then things break/stop working because the way everything works doesn't expect state to mutate in place